### ISSUE TITLE
CSS

### ISSUE THREAD

**Issue by jlag34** [paulpari-s/dasmotos#3]
You did a great job on your CSS. I like that you put your font family on the html so that all of the children elements inherit it. I also seems like you have a good understanding of classes vs ids. One suggestion is to create a class for your price elements instead of using `span`. If it was a larger code base or if you continued to work on this it could lead to unwanted consequences.

**Comment by paulpari-s** [comment]
Thanks for all your comments @jlag34! 
Concerning the `span`, isn't it the only `tag` we can use when we try to style inline instead of by paragraph? Can you otherwise tell me how it should have been done with a class? 
Thanks in advance!

**Comment by jlag34** [comment]
Yeah, for this it's fine. But say if you continued to work on this codebase and it got much larger you would most likely have to change it.
